app/views/organizations/adoptable_pets/show.html.erb

```diff
@@ -154,6 +156,21 @@
                     <%= image_tag('red_no.png', height: '21') %>
                   <% else %>
                     <%= image_tag('green_check.png', height: '21') %>
+                    <% if 'awaiting_review' == @adoption_application.status %>
+                        <span class="d-inline-block" tabindex="0" 
+                              data-bs-toggle="tooltip" title="Please make sure you fill out the adoption questionnaire so staff have enough information to process your application.">
+                          <i class="bi bi-info-circle fs-4 text-secondary ps-1"></i>
+                        </span>
+
+                      <script>
```

Comment:
Very new to Bootstrap so if this `script` tag and the `stylesheet` link on line 1 are unnecessary, please let me know. Still learning!